app/controllers/UserController.scala

```diff
@@ -4,12 +4,23 @@ import auth.AccessScopes.UserReadSelfSecure
 import auth.AuthorisedAction
 import com.gu.identity.auth.AccessScope
 import play.api.*
+import play.api.libs.json.Json
+import play.api.libs.json.Json.toJson
 import play.api.mvc.*
+import services.UserService
+
+import scala.concurrent.ExecutionContext
 
 class UserController(
     val controllerComponents: ControllerComponents,
-    authorisedAction: List[AccessScope] => AuthorisedAction
-) extends BaseController {
+    authorisedAction: List[AccessScope] => AuthorisedAction,
+    userService: UserService
+)(implicit ex: ExecutionContext)
+    extends BaseController {
 
-  def me(): Action[AnyContent] = authorisedAction(List(UserReadSelfSecure))(NotImplemented("TODO: implement me"))
+  def me(): Action[AnyContent] = authorisedAction(List(UserReadSelfSecure)).async(request =>
```

Comment:
I think I might have missed it in a previous PR but I noticed we are using an idapi scope:  
```guardian.identity-api.user.read.self.secure```

Was it a deliberate decision to have gatehouse use idapi scopes instead of defining its own ?

Reply:
Now that I think about it..  What would the final form of this be ? 
Would it be possible to just pass the access token directly to any okta api calls we need ? Or would we have to validate the access token and then use admin level okta credentials to actually access the okta api  ?

Reply:
As far as scopes are concerned, would probably be better to build some gatehouse-specific ones and then delete the Identity API ones so that we know they're not being used.  I'll create a new scope and then change the name here as a follow-up PR.

For the calls to Okta, haven't really got as far as that.  That's why building an update method that calls on Postgres and Okta would be very useful to get a detailed pattern worked out.